common/index.html

```diff
@@ -8,14 +8,14 @@
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
 
-  <script src="/dll.vendor.js"></script>
-  <link rel="manifest" href="/manifest.json">
-  <link rel="shortcut icon" href="/favicon.ico">
-  <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
-  <link rel="icon" type="image/png" href="/favicon/favicon-32x32.png" sizes="32x32">
-  <link rel="icon" type="image/png" href="/favicon/favicon-16x16.png" sizes="16x16">
-  <link rel="mask-icon" href="favicon/safari-pinned-tab.svg" color="#0e97c0">
-  <meta name="msapplication-config" content="/favicon/browserconfig.xml">
+  <script src="./dll.vendor.js"></script>
+  <link rel="manifest" href="./manifest.json">
+  <link rel="shortcut icon" href="./favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="./favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" href="./favicon/favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="./favicon/favicon-16x16.png" sizes="16x16">
+  <link rel="mask-icon" href="./favicon/safari-pinned-tab.svg" color="#0e97c0">
+  <meta name="msapplication-config" content="./favicon/browserconfig.xml">
```

Comment:
These changes will break the production site build, if you land on `myetherwallet.com/account` these will load `/account/dll.vendor...` etc. We'll need to find a way to link these using publicPath.